modules/theme-tools/social-links.php

```diff
@@ -122,7 +122,7 @@ public function customize_register( $wp_customize ) {
 			'priority' => 35,
 		) );
 
-		foreach ( array_keys( $this->publicize->get_services() ) as $service ) {
+		foreach ( array_keys( $this->publicize->get_services( 'connected' ) ) as $service ) {
```

Comment:
Based on the extended description of 3bbf0603784f5eab435316e49cb7d2fb64f88abe and [fallback behavior of an empty argument](https://github.com/Automattic/jetpack/blob/641796c0585718f2f0725e020ee461a03be3e1e6/modules/publicize/publicize-jetpack.php#L358-L360), I'd expect this should be specified as `'all'`, not `'connected'`.